packages/espressocash_app/lib/features/profile/widgets/danger_section.dart

```diff
@@ -22,8 +22,9 @@ class DangerSection extends StatelessWidget {
             label: context.l10n.signOut,
             onPressed: () => _handleLogoutPressed(context),
             color: CpColors.primaryColor,
```

Comment:
This color is wrong. Text color is `E85C30` and chevron color is `F5734A`.

Reply:
Fixed at [359fd3a](https://github.com/espresso-cash/espresso-cash-public/pull/1461/commits/359fd3a2b3c4755698922053729b781ac844dc91)